catalog/core/catalog-core-directorymonitor/src/test/resources/exam.properties

```diff
@@ -1 +1 @@
-pax.exam.karaf.version=${karaf.version}
+pax.exam.karaf.version=4.1.1
```

Comment:
Does property replacement not work here? We should use the variable if at all possible.